lib/logstash/inputs/rabbitmq.rb

```diff
@@ -155,6 +155,10 @@ class RabbitMQ < LogStash::Inputs::Threadable
       # to declare the exchange if it does not exist.
       config :exchange_type, :validate => :string
 
+      # Additional arguments for exchange creation,
+      # for example, alternate-exchange
+      config :exchange_arguments, :validate => :array, :default => {}
```

Comment:
Consistency problem: You called this `arguments` in the output, but `exchange_arguments` here. 

Also, same comments as in the output -- this is validate array but your default is a hash. Further, the docs aren't helpful enough, in my opinion.